.github/scripts/test-coverage/coverage-report

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -euo pipefail
+
+committed_slices="$1"
+installed_slices="$2"
+output_file="${3:-coverage-report.json}"
+
+# Convert to sorted, unique lines
+committed=$(echo "$committed_slices" | tr ' ' '\n' | sort | uniq)
+installed=$(echo "$installed_slices" | tr ' ' '\n' | sort | uniq)
+
+# Save to temp files
+echo "$committed" > /tmp/committed.txt
```

Comment:
i don't like all the hardcoded temp files in here. i 've just bashed my head against the wall for about 1/2h just to find that my tests were picking up phantom slices from stale test files. at the very least i would use `mktemp` and ideally no files at all.